hostdb/hostdb.go

```diff
@@ -62,17 +66,80 @@ type Interaction struct {
 	Result    json.RawMessage
 }
 
+func (i Interaction) IsScan() bool {
+	return i.Type == InteractionTypeScan
+}
+
 // A Host pairs a host's public key with a set of interactions.
 type Host struct {
 	PublicKey     consensus.PublicKey
 	Announcements []Announcement
 	Interactions  []Interaction
 }
 
+func (h *Host) IsHost(host string) bool {
```

Comment:
This seems like a very vague method. Why do we need to have that sort of loose comparison?

Reply:
`blacklist` and `whitelist` - I renamed this a couple of times, the previous name was `CorrespondsWith`.. naming is hard